### ISSUE TITLE
Citation number excluding personal citations

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#691]
From https://rt.inspirehep.net/Ticket/Display.html?id=506650 one can see there is a need for citation count excluding references from papers where the current author is involved. 

Ideally this could be done for authenticated users having cleaned their author profiles and using information stored in the future to come Neo4J.

**Comment by kaplun** [comment]
CC @jmartinm, @tomaszgy, @eamonnmag.

**Comment by eamonnmag** [comment]
Just to confirm, this is to add a field to the page to show the citation count excluding self-citations, rather than changing the actual displayed citation count? I would say this should only be shown on the author profile page itself. Otherwise there's a risk that it will just confuse.

**Comment by kaplun** [comment]
Yes, :+1:  it's more related to the author profile page rather than search results. Sorry if I have not mentioned that part. The important bit here, is really to exclude the citation involving the particular current author (and not in general self citation as defined as "those between papers having non-empty intersection of the corresponding author list").

**Comment by eamonnmag** [comment]
Cool, on the author page, it makes a lot of sense!

**Comment by jmartinm** [comment]
I think this is still relevant, to be moved to a milestone related to author profile development.
